service/src/main/java/com/kodemon/service/implementations/GymServiceImpl.java

```diff
@@ -0,0 +1,254 @@
+
+    @Inject
+    private TrainerDao trainerDao;
+
+    @Inject
+    private PokemonDao pokemonDao;
+
+    @Override
+    public void initializeGyms() {
+        Pokemon pokemon01 = new Pokemon(PokemonName.GEODUDE);
+        Pokemon pokemon02 = new Pokemon(PokemonName.OMANYTE);
+        Pokemon pokemon03 = new Pokemon(PokemonName.RHYHORN);
+        Pokemon pokemon04 = new Pokemon(PokemonName.GOLEM);
+        Pokemon pokemon05 = new Pokemon(PokemonName.ONIX);
+
+        Pokemon pokemon11 = new Pokemon(PokemonName.GOLDEEN);
+        Pokemon pokemon12 = new Pokemon(PokemonName.STARYU);
+        Pokemon pokemon13 = new Pokemon(PokemonName.SEADRA);
+        Pokemon pokemon14 = new Pokemon(PokemonName.WARTORTLE);
+        Pokemon pokemon15 = new Pokemon(PokemonName.GOLDUCK);
+        Pokemon pokemon16 = new Pokemon(PokemonName.GYARADOS);
+
+        Pokemon pokemon21 = new Pokemon(PokemonName.MAGNEMITE);
+        Pokemon pokemon22 = new Pokemon(PokemonName.ELECTRODE);
+        Pokemon pokemon23 = new Pokemon(PokemonName.JOLTEON);
+        Pokemon pokemon24 = new Pokemon(PokemonName.RAICHU);
+        Pokemon pokemon25 = new Pokemon(PokemonName.ELECTABUZZ);
+
+        Pokemon pokemon31 = new Pokemon(PokemonName.GLOOM);
+        Pokemon pokemon32 = new Pokemon(PokemonName.TANGELA);
+        Pokemon pokemon33 = new Pokemon(PokemonName.PARASECT);
+        Pokemon pokemon34 = new Pokemon(PokemonName.VICTREEBEL);
+        Pokemon pokemon35 = new Pokemon(PokemonName.VENUSAUR);
+
+        Pokemon pokemon41 = new Pokemon(PokemonName.EKANS);
+        Pokemon pokemon42 = new Pokemon(PokemonName.KOFFING);
+        Pokemon pokemon43 = new Pokemon(PokemonName.NIDORAN_F);
+        Pokemon pokemon44 = new Pokemon(PokemonName.GOLBAT);
+        Pokemon pokemon45 = new Pokemon(PokemonName.MUK);
+        Pokemon pokemon46 = new Pokemon(PokemonName.NIDOKING);
+
+        Pokemon pokemon51 = new Pokemon(PokemonName.DROWZEE);
+        Pokemon pokemon52 = new Pokemon(PokemonName.ABRA);
+        Pokemon pokemon53 = new Pokemon(PokemonName.HYPNO);
+        Pokemon pokemon54 = new Pokemon(PokemonName.MRMIME);
+        Pokemon pokemon55 = new Pokemon(PokemonName.ALAKAZAM);
+
+        Pokemon pokemon61 = new Pokemon(PokemonName.PONYTA);
+        Pokemon pokemon62 = new Pokemon(PokemonName.CHARMELEON);
+        Pokemon pokemon63 = new Pokemon(PokemonName.FLAREON);
+        Pokemon pokemon64 = new Pokemon(PokemonName.ARCANINE);
+        Pokemon pokemon65 = new Pokemon(PokemonName.MAGMAR);
+        Pokemon pokemon66 = new Pokemon(PokemonName.CHARIZARD);
+
+        Pokemon pokemon71 = new Pokemon(PokemonName.SANDSLASH);
+        Pokemon pokemon72 = new Pokemon(PokemonName.CUBONE);
+        Pokemon pokemon73 = new Pokemon(PokemonName.MAROWAK);
+        Pokemon pokemon74 = new Pokemon(PokemonName.GRAVELER);
+        Pokemon pokemon75 = new Pokemon(PokemonName.DUGTRIO);
+        Pokemon pokemon76 = new Pokemon(PokemonName.RHYDON);
+
+        pokemonDao.save(pokemon01);
```

Comment:
Saving will crash, because you do not set levels, therefore all levels are 0 and therefore it will crash, because there is `@Min(1)` annotation.